docs/computer-vision/object-detection.mdx

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
These empty pages are not shown in the sidebar

Reply:
imo we just don't include them then. The CV docs will eventually be moved under Bot Sequences anyway, so this seems like it's setting us up for confusion later on